openQA.spec

```diff
@@ -63,6 +63,8 @@ BuildRequires:  rubygem(sass)
 Requires:       dbus-1
 Requires:       perl(Minion) >= 9.09
 Requires:       perl(Mojo::RabbitMQ::Client) >= 0.2
+Requires:       perl(JSON::Validator)
```

Comment:
I assume this is required also to run the test suite - so these requires should be part of t_requires

Reply:
This is apparently a leftover from #2001 that wasn't removed when I re-based. Thanks for catching it!